nf_core/pipeline-template/bin/check_samplesheet.py

```diff
@@ -113,19 +113,16 @@ def validate_unique_samples(self):
         """
         Assert that the combination of sample name and FASTQ filename is unique.
 
-        In addition to the validation, also rename the sample if more than one sample,
-        FASTQ file combination exists.
+        In addition to the validation, also rename all samples to have a suffix of _T{n}, where n is the
+        number of times the same sample exist, but with different FASTQ file(s).
```

Comment:
```suggestion
        number of times the same sample exist, but with different FASTQ files, e.g., multiple runs per experiment.
```

Reply:
No. The idea is to always add a `T*` suffix even if there is one sample. That way it allows us to assume everything after the last "_" needs to be trimmed off [here](https://github.com/nf-core/rnaseq/blob/89bf536ce4faa98b4d50a8ec0a0343780bc62e0a/workflows/rnaseq.nf#L199)